measurements/api/measurements.py

```diff
@@ -261,50 +267,13 @@ def get_measurement(measurement_id, download=None):
     return response
 
 
-def input_cte(input_, domain, test_name):
-    """Given a domain or an input_, build a WHERE filter
-    """
-    if input_ and domain:
-        raise BadRequest("Must pick either domain or input")
-
-    if not input_ and not domain:
-        return None
-
-    where_or = []
-    if input_:
-        where_or.append(text("input.input LIKE :i").bindparams(i="%{}%".format(input_)))
-
-    else:
-        domain_filter = "{}%".format(domain)
-        where_or.append(
-            text("input.input LIKE :domain_filter").bindparams(
-                domain_filter=domain_filter
-            )
-        )
-        if test_name in [None, "web_connectivity", "http_requests"]:
-            where_or.append(
-                text("input.input LIKE :http_filter").bindparams(
-                    http_filter="http://{}".format(domain_filter)
-                )
-            )
-            where_or.append(
-                text("input.input LIKE :https_filter").bindparams(
-                    https_filter="https://{}".format(domain_filter)
-                )
-            )
-
-    url_q = (
-        select([column("input").label("input"), column("input_no").label("input_no")])
-        .where(or_(*where_or))
-        .select_from(sql.table("input"))
-    )
-
-    return url_q.cte("input_cte")
-
-
 def log_query(log, q):
     import sqlparse  # debdeps: python3-sqlparse
 
+    if "Select" in repr(type(q)):  # UGLY
```

Comment:
Why is this being done?

Maybe we can add a comment here to explain it.

Perhaps it's possible to rather do something like `isinstance()` or `issubclass`

Reply:
Removed entirely :)